src/core/position.js

```diff
@@ -238,15 +244,22 @@ PROTOTYPE.reposition.offset = function(elem, pos, container) {
 			pos.left -= parentOffset.left + (parseFloat($.css(parent, 'marginLeft')) || 0);
 			pos.top -= parentOffset.top + (parseFloat($.css(parent, 'marginTop')) || 0);
 
+			if (parent.tagName.toLowerCase() === 'BODY') { break; }
```

Comment:
this won't ever happen, cause BODY.toLowerCase() won't be BODY

Reply:
Can we just delete the line?

Reply:
Feel free :) it doesnt work